src/services/useDisplayConfig/index.ts

```diff
@@ -87,14 +108,29 @@ export function useDisplayConfig(
   const displayConfig = ref<DisplayConfig>()
   const displays = ref<DisplayConfig[]>()
 
+  const response = ref<ActionsResponse>()
+
   watchEffect(async () => {
```

Comment:
Maybe add a brief comment on why the fetch is separated from the interpretation of the response?